packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -95,6 +165,7 @@ touch $RPM_BUILD_ROOT/%{cache_dir}/state
 %config(noreplace) %ghost %{conf_dir}/ecs.config
 %config(noreplace) %ghost %{conf_dir}/ecs.config.json
 %ghost %{cache_dir}/ecs-agent.tar
```

Comment:
Should this remain `%ghost`?  It looks like we're now placing a symlink at this path.

Reply:
Yes, this allows the package to own this file without needing to distribute the file (which is symlinked on install).